object_preview-01.js test times out

Noticed this while working on #605 where `Test stripe 4` kept [failing](https://github.com/RecordReplay/devtools/runs/1087529234?check_suite_focus=true). The actual error is:

```
##[error]Failure object_preview-01.js https://replay.io/view?id=ba85c7fc-7d7c-45b3-aa2d-bb2c78feda26
```

The same test seems to fail on master when run locally with `webpack; node test/run.js --pattern object_preview-01.js`.

--- a/src/devtools/client/debugger/packages/devtools-reps/src/reps/prop-rep.js
+++ b/src/devtools/client/debugger/packages/devtools-reps/src/reps/prop-rep.js
@@ -50,11 +50,11 @@ function PropRep(props) {
     if (!suppressQuotes) {
       name = maybeEscapePropertyName(name);
     }
-    key = span({ className: "nodeName", key: `nodeName-${name}` }, name);
+    key = span({ className: "nodeName", key: `nodeName` }, name);
   } else {
     key = Rep({
       ...props,
-      key: `nodeName-${name}`,
+      key: `nodeName`,
       className: "nodeName",
       object: name,
       mode: mode || MODE.TINY,
@@ -67,11 +67,11 @@ function PropRep(props) {
     span(
       {
         className: "objectEqual",
-        key: `objectEqual-${name}`,
+        key: `objectEqual`,
       },
       equal
     ),
-    span({ key: `rep-${name}` }, Rep({ ...props })),
+    span({ key: `rep` }, Rep({ ...props })),
   ];
 }
 

